.github/canary-scale-config.yml

```diff
@@ -343,3 +343,27 @@ runner_types:
     os: linux
     variants:
       ephemeral: {}
+  c.linux.8xlarge.amd:
+    disk_size: 200
+    instance_type: m7a.8xlarge
+    is_ephemeral: true
+    os: linux
+    variants:
+      ephemeral:
+        is_ephemeral: true
```

Comment:
I believe the variants configuration is not needed since it matches the is_ephemeral configuration. Variants are for creating additional runners that have slightly different configuration than the main one.

```suggestion
```